packages/lambdas/src/patient-import-create.ts

```diff
@@ -81,7 +65,9 @@ export async function handler(event: SQSEvent) {
       capture.error(errorMsg, {
         extra: { event, context: lambdaName, error },
       });
-      throw new MetriportError(errorMsg, error, { ...parsedBody });
+      throw new MetriportError(errorMsg, error, {
+        ...{ ...parsedBody, patientPayload: undefined },
```

Comment:
for later: why setting it undefined?

Reply:
Can't be handled in `MetriportError`... and wasn't sure if sending in non-primitive will break things, so just skipped for now.

Reply:
Will handle in v1.

Reply:
https://github.com/metriport/metriport/pull/2872